logstash-core/spec/logstash/queue_factory_spec.rb

```diff
@@ -72,9 +74,10 @@
   context "when `queue.type` is `memory`" do
     before do
       settings.set("queue.type", "memory")
+      settings.set("pipeline.batch.size", 1024)
```

Comment:
@jordansissel because now the size of the `ArrayBlockingQueue` is derived from this setting by the `queue_factory`